src/plot_styles.py

```diff
@@ -58,9 +58,18 @@ def get_system_preferred_style_path(self):
     return stylepath
 
 
+def get_current_style(self):
+    if not self.parent.plot_settings.use_custom_plot_style:
+        style = "adwaita"
+        if Adw.StyleManager.get_default().get_dark():
+            style += "-dark"
+    else:
+        style = self.parent.plot_settings.custom_plot_style
+    return style
```

Comment:
We're kinda having duplicate code here. I'd suggest shortening this function to
`return Path(get_preferred_style_path(self)).name`
or using this directly in line 450

Reply:
I was actually looking for a clean way to get the style, but ended up writing a function instead when I couldn't really figure it out. Now put a variant of this one around line 450 instead.